modules/nodes-scheduler-descheduler-config-map.adoc

```diff
@@ -15,6 +15,7 @@ You should have created a desceduler policy file.
 
 Run the following command to create the configuration map:
 
+[source,bash]
```

Comment:
This is not needed.

Reply:
@ravisantoshgudimetla I removed this file from docs.